src/overlays/actors/ovl_En_Fz/z_en_fz.c

```diff
@@ -85,96 +154,870 @@ static DamageTable D_8093477C = {
+    s16 i;
+    EnFzStruct* ptr = this->unk_23C;
+    Vec3f sp64;
+
+    for (i = 0; i < ARRAY_COUNT(this->unk_23C); i++, ptr++) {
+        if (ptr->unk_00 != 0) {
+            ptr->unk_04.x += ptr->unk_10.x;
+            ptr->unk_04.y += ptr->unk_10.y;
+            ptr->unk_04.z += ptr->unk_10.z;
+
+            ptr->unk_01++;
+
+            ptr->unk_10.x += ptr->unk_1C.x;
+            ptr->unk_10.y += ptr->unk_1C.y;
+            ptr->unk_10.z += ptr->unk_1C.z;
+
+            if (ptr->unk_00 == 1) {
+                if (ptr->unk_2E == 0) {
+                    ptr->unk_2C += 10;
+                    if (ptr->unk_2C >= 100) {
+                        ptr->unk_2E++;
+                    }
+                } else {
+                    ptr->unk_2C -= 3;
+                    if (ptr->unk_2C <= 0) {
+                        ptr->unk_2C = 0;
+                        ptr->unk_00 = 0;
+                    }
+                }
+            } else if (ptr->unk_00 == 2) {
+                Math_ApproachF(&ptr->unk_30, ptr->unk_34, 0.1f, ptr->unk_34 / 10.0f);
+                if (ptr->unk_2E == 0) {
+                    if (ptr->unk_01 >= 7) {
+                        ptr->unk_2E++;
+                    }
+                } else {
+                    ptr->unk_10.x *= 0.75f;
+                    ptr->unk_1C.y = 2.0f;
+                    ptr->unk_10.z *= 0.75f;
+                    ptr->unk_2C -= 17;
+                    if (ptr->unk_2C <= 0) {
+                        ptr->unk_2C = 0;
+                        ptr->unk_00 = 0;
+                    }
+                }
+
+                if ((this->unk_BD9 == 0) && (ptr->unk_2C > 100) && (ptr->unk_38 != 0)) {
+                    this->collider3.dim.pos.x = ptr->unk_04.x;
+                    this->collider3.dim.pos.y = ptr->unk_04.y;
+                    this->collider3.dim.pos.z = ptr->unk_04.z;
+                    CollisionCheck_SetAT(globalCtx, &globalCtx->colChkCtx, &this->collider3.base);
+                }
+
+                sp64.x = ptr->unk_04.x;
+                sp64.y = ptr->unk_04.y + 10.0f;
+                sp64.z = ptr->unk_04.z;
+
+                if ((ptr->unk_2E != 2) && func_809328A4(this, &sp64)) {
+                    ptr->unk_2E = 2;
+                    ptr->unk_10.x = 0.0f;
+                    ptr->unk_10.z = 0.0f;
+                }
+            }
+        }
+    }
+}
+
+void func_80934464(EnFz* this, GlobalContext* globalCtx) {
+    GraphicsContext* gfxCtx = globalCtx->state.gfxCtx;
+    s16 i;
+    u8 flag = 0;
+    EnFzStruct* ptr = this->unk_23C;
+
+    OPEN_DISPS(gfxCtx);
+
+    func_8012C2DC(globalCtx->state.gfxCtx);
+
+    gDPSetColorDither(POLY_XLU_DISP++, G_CD_BAYER);
+    gDPSetAlphaDither(POLY_XLU_DISP++, G_AD_PATTERN);
+
+    for (i = 0; i < ARRAY_COUNT(this->unk_23C); i++, ptr++) {
+        if (ptr->unk_00 > 0) {
+            gDPPipeSync(POLY_XLU_DISP++);
+
+            if (flag == 0) {
+                gSPDisplayList(POLY_XLU_DISP++, &D_04051180);
```

Comment:
These come from gameplay_keep I think, so you can replace them with the appropriate DList from that object. See my object_gmo PR for an example.